<?xml version="1.0"?>
<launch>
 
<param name="robot_description_board" command="$(find xacro)/xacro '$(find franka_cal_sim)/xacro/aprilgrid.xacro'" />



<node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

<include file="$(find franka_cal_sim)/launch/franka.launch">
</include>

<!-- Spawn the aprilgrid into Gazebo -->
<node name="spawn_urdf_board" pkg="gazebo_ros" type="spawn_model" args="-param robot_description_board -urdf -model aprilgrid" />

<node name="image_view" pkg="image_view" type="image_view" args="image:=/simple_camera/image_raw"/>


<node pkg="nodelet" type="nodelet" name="simulation_server" args="load franka_cal_sim/simulation_server standalone_nodelet" output="screen">
</node>


<node pkg="nodelet" type="nodelet" name="model_client" args="load franka_cal_sim/model_client standalone_nodelet" output="screen">
</node>

</launch>
